fix: recursively highlight (meta-)relations & notes

yinanazhou opened the pull request:
- When highlight a parent, all the children get highlighted as well
- Added recursive fundtion `addHoverClassToChildren` and `removeHoverClassToChildren`

refs: #287

yrammos commented on src/js/draw.js:
```diff
@@ -522,3 +499,62 @@ export function draw_metarelation(draw_context, mei_graph, g_elem) {
+  }
+}
+
+// Recursively add hover class to elements and their children
```
Typo: "Recursively ~~add~~ remove class ~~to~~ from elements and their children."

yrammos reviewed (changes requested):
Small typo in a comment—otherwise this looks good to merge. I appreciate the improved CSS filters.

yinanazhou committed:
fix: recursively highlight (meta-)relations & notes

- When highlight a parent, all the children get highlighted as well
- Added recursive fundtion `addHoverClassToChildren` and `removeHoverClassToChildren`

refs: #287
